rmgweb/database/views.py

```diff
@@ -590,6 +1039,284 @@ def parseSoluteDataComment(comment):
+
+
+def get_solvation_solute_data(solute_smiles, solute_estimator, solvent, energy_unit):
+    """
+    Returns a dictionary with solute parameter data for a given list of solute SMILES.
+    If a solvent is selected, solvation properties are also given.
+    """
+    solute_smiles_list = solute_smiles.split()
+
+    solute_data_results = {'Input SMILES': [],
+                           'Error': [],
+                           'E': [],
+                           'S': [],
+                           'A': [],
+                           'B': [],
+                           'L': [],
+                           'V': [],
+                           'Comment': []}
+
+    solute_data_list = []
+    solute_data_found_list = []  # a list of Boolean for whether the solute parameters E, S, A, B, L are all found. (not V)
+    additional_info_list = []  # a list of strings containing additional information to display in the result page
+
+    database.load('solvation')
+    db = database.get_solvation_database('', '')
+
+    # Add epistemic uncertainty columns for SoluteML model
+    if solute_estimator == 'SoluteML':
+        for solute_param_name in ['E', 'S', 'A', 'B', 'L']:
+            solute_data_results[f'{solute_param_name} epi. unc.'] = []
+        additional_info_list.append('epi. unc.: epistemic uncertainty of the SoluteML model.')
+    elif solute_estimator == 'SoluteGC':
+        additional_info_list.append('Comment: functional groups used to estimate the solute parameters')
+
+    # get predictions for each given SMILES
+    for smiles in solute_smiles_list:
+        # initialize the parameter values
+        Error = '-'
+        Comment = '-'
+        solute_data = SoluteData()
+        try:
+            solute_spc = Species().from_smiles(smiles)
+            solute_spc.generate_resonance_structures()
+        except:
+            solute_spc = None
+            Error = 'Unable to parse the SMILES'
+        # Get predictions using the selected method
+        if solute_estimator == 'expt':
+            if solute_spc is not None:
+                data = db.get_solute_data_from_library(solute_spc, db.libraries['solute'])
+                if data is not None:
+                    solute_data = data[0]
+                    Comment = f'From RMG-database solute library: {data[2].index}. {data[2].label}'
+                else:
+                    Comment = 'Not found in RMG-database'
+        elif solute_estimator == 'SoluteGC':
+            if solute_spc is not None:
+                try:
+                    solute_data = db.get_solute_data(solute_spc, skip_library=True)
+                    Comment = solute_data.comment
+                except:
+                    Error = 'Unable to get the prediction from the SoluteGC method'
+        elif solute_estimator == 'SoluteML':
+            solute_epi_unc_dict = {}
+            try:
+                avg_pre, epi_unc, valid_indices = SoluteML_estimator([[smiles]])
```

Comment:
This line always fails. Thus, SoluteML as the solute parameter estimation method always fails to parse the SMILES; for instance, try CC. 
`NameError: name 'rdNormalizedDescriptors' is not defined`
Probably you have to import something else from RDKit.

Reply:
One note (for anyone reading this PR) - this error was obscured because the "except" block does not specify what exception to catch.
Thus, when we had a NameError, this too was treated normally as if there were no problem.
It is generally good practice to specify the exception we expect to catch. The problem I guess in this case is that it's not really obvious what exceptions we should expect.

Reply:
ah I know why it fails. It's because we need to install one more package that cannot be installed through `environment.yml`.
Sorry, I forgot to add that.

You have to install one more package using the following line:
`pip install git+https://github.com/bp-kelley/descriptastorus`
But this is not available through conda install, so it cannot be added to `environment.yml`. Any idea how to add this information to this PR? Other git repo like `chemprop` just mentions to run that line when installing `chemprop`.

Reply:
Regarding your second comment:
I actually have no idea what kind of exception to expect from `SoluteML_estimator`. I have never encountered any error except that name error. I can add the case for `NameError` exception. But besides that, we may never encounter any other errors, so I will just make the rest as general exception.

Reply:
(1) Perhaps we can add it to the .README and to [the wiki](https://github.com/ReactionMechanismGenerator/RMG-Py/wiki/Setting-up-RMG-website). Other than that I am not sure what would be best. I would imagine this is an easy step to miss
(2) I agree that it's not clear what exceptions to expect during "normal operation," and it's probably fine that we just leave it as 'except:'. I did see that if we remove the `except` block we encounter an  `ArgumentError` if the SMILES string cannot be parsed. And then there might be `IndexError` indexing into  `epi_unc`.

Reply:
I was getting `ValueError` instead of `NameError` you got when I uninstalled the `descriptastorus` package to reproduce the error you got. Therefore, I instead catch this type of error by looking for some keywords in the error message as shown in this commit 4b8ca68 and display the following error page with installation instruction:

![descriptastorus_error_msg](https://user-images.githubusercontent.com/17207530/131521784-0665e676-cb9a-400d-821a-ee2639df0c82.png)
 
I also added the installation guide to the .README and to the wiki.